data.js

```diff
@@ -158,7 +158,7 @@ var data = [
     { label: 'MC', value: 'Measurement Canada'},
     { label: 'MC', value: 'Memorandum to Cabinet', explanation: 'The MC focuses on the policy rationale for a new or renewed policy or program initiative'},
     { label: 'MGERC', value: 'Military Grievances External Review Committee'},
-    { label: 'Micromission', value: '3-6 months work assignment. Micromissionee usually comes from other GC department ie secondment'},
+    { label: 'Micromission', value: '3-6 months work assignment. Micromission usually comes from other GC department ie secondment'},
```

Comment:
actually the person coming on a Micromission is called a Micromissionee 😸

Reply:
That's .... a terrible name, heh. Will fix.

Reply:
Better?